.circleci/config.yml

```diff
@@ -2,10 +2,10 @@ version: 2
 jobs:
   build:
     docker: # use the docker executor type; machine and macos executors are also supported
-      - image: circleci/rust:1.48
-      - image: redis:4.0.1
+      - image: circleci/rust:1.57
```

Comment:
Minimum supported rust version changing should also be reported in the changelog

Reply:
Good point!  Thanks Lily!

Reply:
I believe it's ready for the review now.  Please review, @lilymara-onesignal.